src/angular/ngWalker.ts

```diff
@@ -195,7 +195,7 @@ export class NgWalker extends Lint.RuleWalker {
     }
     const sf = ts.createSourceFile(path, `\`${content}\``, ts.ScriptTarget.ES5);
     const original = sf.getFullText;
-    sf.getFullText = function () {
+    sf.getFullText = function() {
```

Comment:
I am usually for white spaces after `function` keyword in anonymous functions. I don't have any specific reasoning, so if you have an argument against it, I am fine :-)